tests/utils/inputparams/testparams.go

```diff
@@ -42,10 +43,10 @@ func init() {
 func GetVolumeName() string {
 	return volumeName
 }
-
 // GetVolumeNameWithTimeStamp prepares unique volume name by appending current time-stamp value
-func GetVolumeNameWithTimeStamp(volumeName string) string {
-	return volumeName + "_volume_" + strconv.FormatInt(time.Now().Unix(), 10)
+func GetVolumeNameWithTimeStamp(volName string) string {
+	return volName + "_volume_" + strconv.FormatInt(rand.Int63(), 10)
```

Comment:
I don't know if any one else noticed but I keep getting the same file name across multiple calls to this function and the tests break because two volume creates use the same name. The timestamp doesn't change between calls. The rand function works better. I'll make a separate function so users of the existing function can continue using that.